### ISSUE TITLE
Elemental shaman recommendation issue

### ISSUE THREAD

**Issue by akshay14021** [Hekili/hekili#3543]
### Before You Begin

- [X] I confirm that I have downloaded the latest version of the addon.
- [X] I am not playing on a private server.
- [X] I checked for an [existing, open ticket](https://github.com/Hekili/hekili/labels/recommendation) for this issue and was not able to find one.
- [X] I edited the title of this issue (above) so that it describes the issue I am reporting.
- [X] I am reporting an issue with the default priority included with the specialization (imported or edited priorities are not supported).

### Describe the Issue

For Elemental shaman, spiritwalker grace , nature swiftness are not recommended and after opener, I mostly see lightening bolt even if cds are available
No totemic recall recommendation
No Earth shield recommendation

### How to Reproduce

Enter game as Elemental shaman, loadout is from wowhead m+
Start moving - no recommendation on spirit walker grace
Before opener - no recommendation on nature swiftness
After opener - mostly recommendation on lightening bolt even if maelstrom procs are ready
No totemic recall recommendation
No Earth shield recommendation

### Snapshot (Link)

https://pastebin.com/KBwex2Ls

### Raidbots Sim Report (Link)

https://www.raidbots.com/simbot/report/stbAhdzWGWEL5Pz5c1ch5p

**Comment by syrifgit** [comment]
Looks like a talent recognition issue with the Farseer hero tree, with ancestral replacing natures swiftness? @Hekili 

`ancestral_swiftness = 1/1`

```
 20.  natures_swiftness ( default - 11 ) - talent [ ancestral_swiftness ] disallowed
        Time spent on this action:  0.01ms
        TimeData:Elemental-default-11:natures_swiftness:0.01:Ability Known, Enabled(0.01)
```

```
 30.  ancestral_swiftness ( single_target - 7 ) - ability unknown
        Time spent on this action:  0.01ms
        TimeData:Elemental-single_target-7:ancestral_swiftness:0.01:Ability Known, Enabled(0.01)
```

**Comment by akshay14021** [comment]
I was on stormbringer tree with nature swiftness on, but now I have removed nature swiftness, will post a new log @syrifgit

**Comment by syrifgit** [comment]
Okay. In that case it seems more like the survival hunter issue where it thinks you have both trees.

**Comment by akshay14021** [comment]
https://pastebin.com/FNwx1gub @syrifgit this is new one - without nature swiftness, currently using stormbringer tree - looks much better now but spiritwalker still not recommended

**Comment by Hekili** [comment]
Spiritwalker's Grace logic depends on distance to your target (which doesn't make a ton of sense, but is the closest equivalent the addon can use in terms of knowing how much movement you'll be doing).

The rest should be solved with next release (v11.0.2-1.0.4).  If not, you can open a new ticket with a snapshot for each specific issue that is still occurring.
